src/components/tree/phyloTree/types.ts

```diff
@@ -0,0 +1,260 @@
+import { NODE_NOT_VISIBLE, NODE_VISIBLE_TO_MAP_ONLY, NODE_VISIBLE } from "../../../util/globals";
+import { PhyloTree } from "./phyloTree";
+
+export type StringLayouts = "scatter"
+
+export type NumericLayouts = "rect" | "radial" | "unrooted" | "clock"
+
+export type Layout = NumericLayouts | StringLayouts
```

Comment:
Question - why split into StringLayouts and NumericLayouts? (and what's the difference?) I don't see either of those variables used elsewhere.

Reply:
Oops, this is leftover from b37a77c9555b11907c6698e6a1d07ab94533d00f. I'll update to `Layout = "rect" | "radial" | "unrooted" | "clock" | "scatter"`

Reply:
Done: https://github.com/nextstrain/auspice/blob/1960d5a3f9dc55a7946c9129ba67732db09081d9/src/components/tree/phyloTree/types.ts#L4